plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -122,7 +122,7 @@ private static String prepareCassandraYaml()
 
         File yamlFile = tmpDirPath.resolve("cu-cassandra.yaml").toFile();
         yamlFile.deleteOnExit();
-        write(modified, yamlFile, UTF_8);
+        asCharSink(yamlFile, UTF_8).write(modified);
```

Comment:
```java
Files.writeString(yamlFile, modified);

Reply:
Updated PR to use Files.writeString / Files.write (byte[]) in place of `asCharSink(...UTF_8)` and `asByteSink()`